src/server/api/AdminPanel.ts

```diff
@@ -0,0 +1,131 @@
+import type { CustomRequest } from "../../types.js";
+import type { Response } from "express";
+// @ts-ignore
+import { deleteUser, getMemberDataByCriteria, updateMemberColumns } from "../database/memberManager.js";
+// @ts-ignore
+import { deleteRefreshTokensOfUser } from "../database/refreshTokenManager.js";
+// @ts-ignore
+import { logEvents } from "../middleware/logEvents.js";
+
+async function processCommand(req: CustomRequest, res: Response): Promise<void> {
```

Comment:
Why does this need to be async?

Reply:
Because it contains an await statement.

Reply:
Nothing pops up when I searchfor "await".. 🤔 Perhaps there used to be but now it's gone?

Reply:
Oh that was the wrong function, I was thinking of `sendCommand` which does have an await statement.